stagehand/context.py

```diff
@@ -40,7 +40,10 @@ async def inject_custom_scripts(self, pw_page: Page):
     async def get_stagehand_page(self, pw_page: Page) -> StagehandPage:
         if pw_page not in self.page_map:
             return await self.create_stagehand_page(pw_page)
-        return self.page_map[pw_page]
+        stagehand_page = self.page_map[pw_page]
+        # Update active page when getting a page
+        # self.set_active_page(stagehand_page)
```

Comment:
we either want this so uncomment or we don't want this so get rid of it? we shouldnt have setter behaviour in a getter method imo